ivy_tests/test_ivy/test_frontends/test_numpy/helpers.py

```diff
@@ -21,9 +21,7 @@ def _test_frontend_function_ignoring_unitialized(*args, **kwargs):
         return
     ret, frontend_ret = values
     ret_flat = [np.where(where, x, np.zeros_like(x)) for x in helpers.flatten(ret=ret)]
-    frontend_ret_flat = [
-        np.where(where, x, np.zeros_like(x)) for x in helpers.flatten(ret=frontend_ret)
-    ]
+    frontend_ret_flat = [np.where(where, x, np.zeros_like(x)) for x in frontend_ret]
```

Comment:
Please do not change the helper functions in general. I have run the tests and spotted the same failure. The reason is because `numpy` returns an `int` output given an `int` input. Please try to handle this special case in either your frontend implementation or your test codes, instead of changing the helper function. Thanks :)